### ISSUE TITLE
Mini metro don't start (bad path)

### ISSUE THREAD

**Issue by Amosar** [flathub/com.valvesoftware.Steam#136]
System information:
OS: Linux mint 19
DE: XFCE
flatpak: Flatpak 0.99.3

I have run ``flatpak update`` and I've the latest version installed

When I try to launch Minimetro I have this error:
https://pastebin.com/SjJPAhA6

To help, this is the launch script of MiniMetro:
https://pastebin.com/jnxMpDFF
And I'm able to launch to the game if I modify the last lines with the absolute path of the game 
```
EXE="/home/amosar/.var/app/com.valvesoftware.Steam/data/Steam/steamapps/common/MiniMetro/Mini Metro.x86_64"
exec "${EXE}"```

**Comment by nanonyme** [comment]
sh_makepath: error retrieving current directory: getcwd: cannot access parent directories: Success

**Comment by Amosar** [comment]
Yes it seems to be the issue, is it a flatpak or a runtime issue?
I'm new with flatpak ^^

**Comment by nanonyme** [comment]
I can't say for sure what the root cause is but I've been seeing the same error in some other logs as well. It would be good to know what causes this as I was never able to replicate this through flatpak run --command=bash com.valvesoftware.Steam . This could explain an entire category of issues if Steam is having difficulties setting current working directory for games.

**Comment by nanonyme** [comment]
Could you please give this another whack? A recent app update juggled quite a lot of paths around

**Comment by jjardon** [comment]
Might be related: https://gitlab.com/freedesktop-sdk/freedesktop-sdk/issues/320

**Comment by nanonyme** [comment]
@jjardon any plan merging the other workarounds? NixOS is the same system we just recently adapted a timezone workaround from

**Comment by nanonyme** [comment]
@Amosar probably no need to re-test until that change mentioned by @jjardon is shipped

**Comment by jjardon** [comment]
@nanonyme @Amosar FYI: the change has been integrated and the new runtimes have been imported to flathub yesterday

**Comment by nanonyme** [comment]
Too bad the imports take around 9 hours to run and do not always pass (eg yesterday)

**Comment by nanonyme** [comment]
@jjardon import failed again. Sounds like import process doesn't really scale anymore

**Comment by jjardon** [comment]
@nanonyme done now: https://flathub.org/builds/#/builders/7/builds/19 I think it was a problem updating to flatpak 1.0
